### DISCUSSION TITLE
Add Depth selection to the skin customiser

### DISCUSSION THREAD

**Discussion by mk56-spn** [ppy/osu#20114] [Ideas]
currently the depth sorting in lazer seems to be an automated system which just tells the next drawable to be one depth above the previous one, or something of the sort, issue is if you want to rework something on a lower layer right now you currently can't, because you cant manually modify drawable depth in skin customisation.

Id like to attempt adding this, unless theres already plans for it.

**Comment by peppy** [comment]
You're welcome to attempt it. I'm not sure how it should be specified (either based on order of passing json, or an actual depth specification?) but that should be decided before any implementation happens.

**Reply by mk56-spn** [reply]
im inclined towards an actual depth specification.

i have other concerns as well, mainly UI based, for one i think that the object depth should be accessible from the right click pop up menu ( the one that houses anchoring and origin  )
